contracts/ServiceRequest.sol

```diff
@@ -279,13 +282,10 @@ contract ServiceRequest {
 
     function closeRequest(uint256 requestId) public returns(bool) {
         
-        // Should be ative foundation Member
-        require(foundationMembers[msg.sender].status);
-        
         Request storage req = requests[requestId];
-        
-        // Request should be active
-        require(req.status == RequestStatus.Approved);  
+
+        // Should be ative foundation Member or Request Owner
```

Comment:
(it seems it was lost in my first "review").
There is a typo in comment :) (ative)